docs/cloud/managing-airbyte-cloud.md

```diff
@@ -186,17 +187,18 @@ In a successful sync, the number of emitted records and committed records should
 
 ## Edit stream configuration
 
-1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click **Connections** and then click a connection in the list you want to change.   
+1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click **Connections** and then click the connection you want to change.   
 
 2. Click the **Replication** tab.
 
-The **Transfer** and **Streams** settings include the following parameters:
+The first section includes the following parameters:
```

Comment:
This sentence implies there's a second section, but I didn't see any info about it in the doc. What are we trying to say here?

Reply:
@Amruta-Ranade I realized that the new wording here reflected the new streams table redesign, which hasn't been implemented yet. I reverted this text back.